experimental/ModStd/ModStdQt.jl

```diff
@@ -567,18 +567,15 @@ algebraic closure of Q(A).
 
 The return value is an array 
 - first entry is a leading coefficient
-- the others are tuples, one for each irreducible factor of the input, either
+- the others are tuples, one for each irreducible factor of the input, namely
 
-  - two polynomials and an integer: the polynomials are 
+  two polynomials and an integer: the polynomials are 
       over a finite extension of Q(A) given as a residue field,
       Q(A)[t]/h, s.th. the first polynomial is an abs. irreducible factor over this
       extension, the last entry (the integer) is the multiplicity.
-      In this case, there are degree(h(t)) many abs. irreducible factors, but they
-      are all conjugate to the 1st tuple entry. The 2nd entry is the product of all the
-      other conjugates,
-
-  - one polynomial over Q(A) - indicating that this factor is abs. irreducible 
-      and the multiplicity
+      Since there are degree(h(t)) many abs. irreducible factors, which 
+      are all conjugate to the 1st tuple entry. 
```

Comment:
```suggestion
      are all conjugate to the first tuple entry. 
```

Reply:
Something is wrong here with this sentence. It starts with "Since"... but then there is no "it follow..." or any other "consequence?!